templates/enduro.toml.j2

```diff
@@ -12,24 +12,32 @@ address = "{{ enduro_listen_temporal_address }}"
 taskQueue = "global"
 {% endif %}
 
+[internalapi]
```

Comment:
This block should be inside enduro_sfa_pipelines, in case we get to merge this back to main someday in the short future.